### PR TITLE
Add tests for FramebufferTextureLayer

### PR DESCRIPTION
**Pull request by qkmiao**

### CODE REVIEW

File: sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+        "attaching a texture to a framebuffer should succeed.");
+    gl.framebufferTextureLayer(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, null, 0, 0);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+        "detaching a texture from a framebuffer should succeed.");
+
+    var maxTexSize = gl.getParameter(gl.MAX_3D_TEXTURE_SIZE);
+    var maxLevels = numLevelsFromSize(maxTexSize);
+    gl.framebufferTextureLayer(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, tex3d, maxLevels - 1, 0);
+    wtu.glErrorShouldBe(gl, gl.NO_ERROR,
```
Comment (zhenyao) [inline sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html]: Can you also checkFramebuffer() here?

**Comment by zhenyao** [comment]
LGTM
Could you upload another pull request to add the test case I suggested? But let me merge this now.

File: sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html (hunk 1 above)
Comment (kenrussell) [inline sdk/tests/conformance2/renderbuffers/framebuffer-texture-layer.html]: Yes, the framebuffer should be verified to be incomplete at this point.

**Comment by kenrussell** [comment]
LGTM after the fact. Thanks @qkmiao for this contribution and @zhenyao for reviewing it.

**Pull request merged**